hudi-common/src/test/java/org/apache/hudi/payload/TestAWSDmsAvroPayload.java

```diff
@@ -18,14 +18,13 @@
 
 package org.apache.hudi.payload;
```

Comment:
by right the testing class package should be the same as the tested class

Reply:
fixed